pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -124,9 +124,9 @@ export class LogsPanel extends React.Component {
 
     render() {
         return (
-            <div className="panel panel-default cockpit-log-panel">
+            <div className="panel panel-default cockpit-log-panel" role="table">
                 <div className="panel-heading">{this.props.title}</div>
```

Comment:
Ideally, this would have an id, and its parent with `role="table"` would use `aria-labelledby` to point at it.